submissions/famuyiwaOlawumi/html-forms/contact.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>contact us</title>
+</head>
+<body>
+    <h1>Contact Form</h1>
+<form action="#" method="post" enctype="multipart/form-data">
```

Comment:
You need to remove attributes `action`, `method` and `enctype`. They are not needed yet. @Olawumi11